config/locales/en.yml

```diff
@@ -57,3 +61,22 @@ en:
       full: "%B %-d, %Y"
       youth_date_of_birth: "%B %Y"
       short_date: "%-m/%d"
+  activerecord:
+    models:
+      judge:
+        one: Judge
+        other: Judges
```

Comment:
Let's leave out `other:` and use https://apidock.com/rails/ActionView/Helpers/TextHelper/pluralize in the code instead